src/main/java/gift/order/controller/api/OrderController.java

```diff
@@ -0,0 +1,64 @@
+package gift.order.controller.api;
+
+import gift.exception.MyException;
+import gift.infra.LoggedInMember;
+import gift.member.entity.Member;
+import gift.order.dto.OrderDetails;
+import gift.order.dto.OrderRequestDto;
+import gift.order.dto.OrderResponseDto;
+import gift.order.service.OrderService;
+import gift.wishlist.service.WishListService;
+import jakarta.validation.Valid;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequestMapping("/api")
+@RestController
+public class OrderController {
+
+    private OrderService orderService;
+    private WishListService wishListService;
+
+    public OrderController(OrderService orderService, WishListService wishListService) {
+        this.orderService = orderService;
+        this.wishListService = wishListService;
+    }
+
+    @Transactional
+    @PostMapping("/orders")
+    public ResponseEntity<OrderResponseDto> createOrder(
+            @RequestBody @Valid OrderRequestDto orderRequestDto,
+            @LoggedInMember Member member
+    ){
+        OrderResponseDto orderResponseDto = orderService.createOrder(orderRequestDto, member.getMemberId());
+        //상품 바로 구매를 하는 경우
+        if(orderRequestDto.wishId() != null){
+            wishListService.removeFromWishList(orderRequestDto.wishId());
+        }
+        return new ResponseEntity<>(orderResponseDto, HttpStatus.CREATED);
```

Comment:
```suggestion
        return ResponseEntity.status(HttpStatus.CREATED)
            .body(orderResponseDto);
```
이렇게하면 가독성이 더 좋아질것 같네요 :)

Reply:
네 해당 방법이 더 가독성이 좋은거 같습니다!! 반영했습니다.